spec/controllers/assets_controller_spec.rb

```diff
@@ -25,13 +25,41 @@
         expect(assigns(:asset).file.path).to match(/asset\.png$/)
       end
 
+      it 'stores access_limited_user_ids as access_limited on asset' do
+        attributes = valid_attributes.merge(access_limited_user_ids: ['user-id'])
+        post :create, params: { asset: attributes }
+
+        expect(assigns(:asset).access_limited).to eq(['user-id'])
+      end
+
       it 'stores access_limited on asset' do
         attributes = valid_attributes.merge(access_limited: ['user-id'])
         post :create, params: { asset: attributes }
 
         expect(assigns(:asset).access_limited).to eq(['user-id'])
       end
 
+      it 'stores access_limited blank string as empty array on access_limited' do
+        attributes = valid_attributes.merge(access_limited: '')
+        post :create, params: { asset: attributes }
+
+        expect(assigns(:asset).access_limited).to eq([])
+      end
+
+      it 'stores access_limited_user_ids blank string as empty array on access_limited' do
+        attributes = valid_attributes.merge(access_limited_user_ids: '')
+        post :create, params: { asset: attributes }
+
+        expect(assigns(:asset).access_limited).to eq([])
+      end
+
+      it 'stores access_limited_organisation_ids on asset' do
```

Comment:
I think we're missing a test for the empty string case with this new attribute.

Reply:
Is it covered by the test case on lines 49-54?